--- PKGBUILD	2018-05-23 15:30:31.239238190 +0800
+++ PKGBUILD	2018-05-23 15:30:38.659279944 +0800
@@ -35,10 +35,15 @@
   cd ${srcdir}/ccache
 
   install -Dm 755 ccache ${pkgdir}/usr/bin/ccache
-  install -Dm 644 ccache.1 -t ${pkgdir}/usr/share/man/man1/ccache.1
+  install -Dm 644 doc/ccache.1 -t ${pkgdir}/usr/share/man/man1
+  install -Dm 644 doc/AUTHORS.adoc ${pkgdir}/usr/share/doc/${pkgname}
+  install -Dm 644 doc/MANUAL.adoc ${pkgdir}/usr/share/doc/${pkgname}
+  install -Dm 644 doc/NEWS.adoc ${pkgdir}/usr/share/doc/${pkgname}
 
   install -d ${pkgdir}/usr/lib/ccache/bin
   ln -sf /usr/bin/ccache ${pkgdir}/usr/lib/ccache/bin/cc 
+  ln -sf /usr/bin/ccache ${pkgdir}/usr/lib/ccache/bin/clang
+  ln -sf /usr/bin/ccache ${pkgdir}/usr/lib/ccache/bin/clang++
   ln -sf /usr/bin/ccache ${pkgdir}/usr/lib/ccache/bin/gcc
   ln -sf /usr/bin/ccache ${pkgdir}/usr/lib/ccache/bin/g++
   ln -sf /usr/bin/ccache ${pkgdir}/usr/lib/ccache/bin/cpp
