diff --git a/scripts/setlocalversion b/scripts/setlocalversion
index 43f29132a28c..a8513d575410 100755
--- a/scripts/setlocalversion
+++ b/scripts/setlocalversion
@@ -48,39 +48,8 @@ scm_version()
 
 		# If we are at a tagged commit (like "v2.6.30-rc6"), we ignore
 		# it, because this version is defined in the top level Makefile.
-		if atag="`git describe --exact-match --abbrev=0 2>/dev/null`"; then
-			# Make sure we're at the tag that matches the Makefile.
-			# If not place the hash of the tag as well for
-			# v2.6.30-rc5-g314aef
-			if [ "x$atag" != "x$VERSION" ]; then
-				# If only the short version is requested,
-				# don't bother running further git commands
-				if $short; then
-					echo "+"
-					return
-				fi
-				printf '%s%s' -g "`git show-ref -s --abbrev --tags $atag 2>/dev/null`"
-			fi
-		else
-
-			# If only the short version is requested, don't bother
-			# running further git commands
-			if $short; then
-				echo "+"
-				return
-			fi
-			# If we are past a tagged commit (like
-			# "v2.6.30-rc5-302-g72357d5"), we pretty print it and
-			# include the hash of any new tag on top.
-			if atag="`git describe 2>/dev/null`"; then
-				tag="`git describe --abbrev=0 2>/dev/null`"
-				commit="`echo "$atag" | awk -F- '{printf("-%05d-%s", $(NF-1),$(NF))}'`"
-				printf '%s%s%s' -g "`git show-ref -s --abbrev --tags $tag 2>/dev/null`" $commit
 			# If we don't have a tag at all we print -g{commitish}.
-			else
 				printf '%s%s' -g $head
-			fi
-		fi
 
 		# Is this git on svn?
 		if git config --get svn-remote.svn.url >/dev/null; then
@@ -88,7 +57,7 @@ scm_version()
 		fi
 
 		# Check for uncommitted changes
-		if git diff-index --name-only HEAD | grep -qv "^scripts/package"; then
+		if git diff-index --name-only HEAD | grep -qv "^scripts/setlocalversion"; then
 			printf '%s' -dirty
 		fi
 
